### ISSUE TITLE
r747, Texplorer, Search Bar... weirdness

### ISSUE THREAD

**Issue by ghost** [ME3Explorer/ME3Explorer#286]
@KFreon -- Just wanted to bring your attention to this. I have no debug output and nothing I can post to be of real help.

I finished a Texplorer scan and tried to use the Search bar and the tool didn't respond. This is similar to what I reported recently where I had the same thing happen, but after a texture replacement. What's strange is that in this case, closing and re-opening Texplorer didn't help. I had to close and reopen _the toolset_. Then it worked again.

I'm not sure if that last bit provides anything of use, but there's definitely some wonkiness going on.

Feel free to close the issue; I'm not sure what can be done with so little information. I just wanted you to know, and for us to have a breadcrumb. If I ever do get any type of error message, I'll re-open this and post, of course.

**Comment by alvarome** [comment]
Hi:

Not the same issue here but I´m also experiencing a very unusual behaviour with this rev.

Yesterday, I opened the toolset, used texplorer and closed both. Next time I opened the toolset Texplorer didn´t launch. TPF/DDS tools neither.

No background tasks running, no nothing.

Closing the toolset didn´t change anything. 

Restarting the system didn´t change anything.

I made it working again copying the toolset to a new folder. But after using the toolset for once, all started again.

I have not reported it as a bug because I have no clue what´s going on.

Two cause candidates here... just to say something.

A.- The evil Windows SO + antimalware process + antivirus combo. I say this because when I downloaded rev746 it was blocked by Chrome, something I did not experience long ago with previous toolset releases and, even more, downloading rev747 was fine. But you never know what or when those more and more agressive background scans are doing in the system and when a file is flaged as suspicious.

In fact, Windows 10 Defender wanted to report Me3Explorer.zip file, something I denied.

B.- The folder at userprofile used by ME3Explorer in case to have some revs working -that´s my case because I´m using rev738 and lastest one and I´m sure I´m not the only one-.

But what´s really strange is that only Texplorer and TPF/DDs tools stopped working. Other toolset tools opens always.

HTH!
Alvaro

EDIT --> Both times I used texplorer I used the search bar!

EDIT --> Not mentioned above but opening the debug window before trying to open Texplorer or TPF/DDS tools revelas... nothing at all.

EDIT --> BTW... the "SEND TO KFREON.txt" file in my \exec folder does not belong to me! I mean... there is such file already in the toolset .zip package...

**Comment by KFreon** [comment]
The Send to me thing is a whoopsy on my part. Forgot to delete it when I uploaded the rev.
All that AV stuff doesn't sound good, and I don't know why it would flag anything in this version compared to stuff in the last version...

I haven't been able to replicate either of these issues, and the only real way to find out what's going on is to run it through VS. :(

**Comment by alvarome** [comment]
Hi:

Above I said I made Texplorer working again -at least for a moment- copying the toolset to a new folder. A nosense but it worked! 

I have just checked that another temporal fix is deleting the rev747 ME3Explorer "user.config" file at userprofile\WV...

-those identified with a 1.2.747.0 folder-

HTH?
Alvaro

PS I have had a Texplorer crash just looking for a texture. I think this has nothing to do with the previous -perhaps just a bad pcc on my side- but as I got one of those feared ;) "Let me know about this" exception windows, here you got data in .txt file:

https://onedrive.live.com/redir?resid=B951C35A69BAEE6D!142&authkey=!ANc-bUMqRo9_mQ0&ithint=file%2ctxt

What I found strange here is that it is refereing to a D:\MyDocuments folder not present in my system! -?-

In other words, my MyDocuments folder is not at D:

**Comment by KFreon** [comment]
@alvarome - interesting discovery with the user.config. Does that fix it forever? @giftfish does it work for you too?

Also, probably stick that last bit into another issue. I don't have access to the code right now, but I remember that shouldn't be able to be hit.
I'm guessing the texture/file was modified?

**Comment by ghost** [comment]
@KFreon -- Will try and let you know. So far I haven't had any crashes since the NET update.

**Comment by alvarome** [comment]
@KF, be ready to read something strange:

Texplorer and TPF/DDS tools works fine here now everytime I use the toolset... until I disable the first time DLC extraction.

When disabling DLC extraction initial window using the "options" --> "advanced" menu, I can still use the toolset during that session but next time I started it neither Texplorer nor TPF/DDS tools works for me... and only Texplorer and TPF/DDS tools. Perhaps because those are more "DLC" extraction dependant apps?

That may explain why deleting the user.config file the functionality is restored -?-. 

Here you got user.config file before disabling DLC extraction:

https://onedrive.live.com/redir?resid=B951C35A69BAEE6D!145&authkey=!AI81FbbSttdvTxA&ithint=file%2ctxt

And here after disabling DLC extraction using the options --> advanced menu.

https://onedrive.live.com/redir?resid=B951C35A69BAEE6D!143&authkey=!AIuYydY3V-VllgY&ithint=file%2ctxt

HTH!
Alvaro

**Comment by alvarome** [comment]
Q here: copying the toolset in a new and different folder means a new key at the userprofile WV folder? If so, that may also explain why it worked then.
Alvaro

**Comment by KFreon** [comment]
I thought they were supposed to be grayed out. That's what I did right @giftfish ?
I'd check but I'm at work.

@alvarome - Likley it would get a new key yes.

**Comment by alvarome** [comment]
And about the Texplorer crash -that I think that perhaps is a problem on my side- yes, that texture was changed in a previous modding run. But I was working now with the vanilla backup.

HTH!
Alvaro

**Comment by ghost** [comment]
@KFreon -- To my knowledge you didn't gray them out, no. You overruled me on that, as far as I know, but let me take a quick look.

**Comment by SirCxyrtyx** [comment]
Not grayed out. They just don't open.

**Comment by ghost** [comment]
Holy cow. Yeah, they just don't open.

I'm processing right now... gimme sec.

**Comment by KFreon** [comment]
Oh, I should definitely have a warning then.
Why would I have done it that way? If I fought to have them available, what's the point if they don't start...

Either way, this doesn't address the actual issue here, that of the search bar crash.

**Comment by ghost** [comment]
@KFreon -- Yeah, I think we can change that, but I'm going to re-open an old issue and put my thoughts there. Otherwise if we need to look at it again in the future, we'll lose track.

Gimme sec. Btw... I can't believe you did that without mentioning it, LOL.

**Comment by ghost** [comment]
FYI, re-opened issue #132 so the disabled tool thing can be addressed. 

TL;DR -- ModMaker can be re-enabled, as can TPF Tools. Texplorer can also, but we have to implement a change to the TexFirstTimeSetup GUI to accomodate it.

**Comment by KFreon** [comment]
I'll comment there about all this then. 

Anything further on the search bar?

**Comment by ghost** [comment]
@KFreon -- I don't, yet, no. I'm done testing for this rev, actually. I could try to rebuild trees a couple more times and see if I can force something to crash, but that's about it.

**Comment by ghost** [comment]
@KFreon -- Hasn't happened again, but I have a feeling we haven't found the problem. I guess it's possible the NET update fixed it as well, but I don't have any further data to give you.

You can close if you want, and I'll re-open if it happens again.

**Comment by KFreon** [comment]
Yeah I'm out of ideas :(

**Comment by ghost** [comment]
This also hasn't reappeared (so far) in rev748.

Will call it verified.
